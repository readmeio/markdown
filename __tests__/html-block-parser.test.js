import { mdast } from '../index';

describe('Parse html block', () => {
  it('parses an html block', () => {
    const text = `
<div>Some block html</div>
    `;

    expect(mdast(text)).toMatchSnapshot();
  });
});
