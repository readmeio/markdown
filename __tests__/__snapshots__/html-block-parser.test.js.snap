// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parse html block parses an html block 1`] = `
Object {
  "children": Array [
    Object {
      "block": true,
      "type": "html",
      "value": "<div>Some block html</div>
    ",
    },
  ],
  "type": "root",
}
`;
